openslides/assignments/migrations/0001_initial.py

```diff
@@ -61,6 +61,7 @@ class Migration(migrations.Migration):
                 ('votescast', openslides.utils.models.MinMaxIntegerField(blank=True, null=True)),
                 ('published', models.BooleanField(default=False)),
                 ('yesnoabstain', models.BooleanField(default=False)),
+                ('yesno', models.BooleanField(default=False)),
```

Comment:
Ah this looks problematic. Users with a database create with OpenSlides 2.0 will get an migration error. I think you have to do the following:
- Revert the changes in this file.
- Run `python manage.py makemigrations`. This will create a new migration file `0002_xyz.py`.
- Commit this file.
- Now you can test everything:
  - Remove old database and try to start OpenSlides with `python manage.py start`.
  - Remove old database. Checkout OpenSlides 2.0 `git checkout 2.0`. Start OpenSlides. Then checkout your branch and try to run `python manage.py migrate`. This should work without errors.